### PR TITLE
Update SchemaFrame styling

### PR DESCRIPTION
**Pull request by nglgzz**
Before:
before
After:
after

Also:
- fixed some typos 
- renamed the default export in `browser/modules/Docs/Docs.jsx` to Docs as it's never imported as Guides

### CODE REVIEW

File: src/browser/modules/Docs/Docs.jsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ import Directives from 'browser-components/Directives'
 import Carousel from '../Carousel/Carousel'
 import Slide from '../Carousel/Slide'
 
-export default class Guides extends Component {
+export default class Docs extends Component {
```
Comment (oskarhane) [inline src/browser/modules/Docs/Docs.jsx]: 👍

File: src/browser/modules/Stream/HelpFrame.jsx [hunk 2]
```diff
@@ -45,7 +45,7 @@ const HelpFrame = ({ frame }) => {
       const { title, subtitle } = chapter
       let { content } = chapter
 
-      // The commands topic is a special case that uses dymaic data
+      // The commands topic is a special case that uses dynamic data
```
Comment (oskarhane) [inline src/browser/modules/Stream/HelpFrame.jsx]: 👍

File: src/browser/modules/Stream/SchemaFrame.jsx [hunk 3]
```diff
@@ -79,49 +129,26 @@ export class SchemaFrame extends Component {
...
-    return `${indexString}\n\n${constraintsString}\n`
-  }
+    const frame = (
+      <Slide>
+        <SchemaTable name='Indexes' content={indexes} />
+        <SchemaTable name='Constraints' content={constraints} />
+        <br />
+        <p className='lead'>Execute the following query for more information</p>
+        <figure>
+          <pre className='code runnable'>CALL db.schema.visualization</pre>
```
Comment (oskarhane) [inline src/browser/modules/Stream/SchemaFrame.jsx]: neo4j-browser needs (at the time of writing this) to support Neo4j 3.4+, and `db.schema.visualization` does not exist in Neo4j 3.4. There it's named `db.schema()`, so you need to do a version check here.

File: src/shared/modules/cypher/cypherDuck.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ export const CLUSTER_CYPHER_REQUEST = NAME + '/CLUSTER_REQUEST'
 export const FORCE_CHANGE_PASSWORD = NAME + '/FORCE_CHANGE_PASSWORD'
 
 // Helpers
-const queryAndResovle = async (driver, action, host, useDb = {}) => {
+const queryAndResolve = async (driver, action, host, useDb = {}) => {
```
Comment (oskarhane) [inline src/shared/modules/cypher/cypherDuck.js]: 😱

File: src/browser/modules/Stream/SchemaFrame.jsx [hunk 5]
```diff
@@ -79,49 +129,26 @@ export class SchemaFrame extends Component {
...
+    const constraints = formatConstraints(this.state.constraints)
 
-    return `${indexString}\n\n${constraintsString}\n`
-  }
+    const frame = (
+      <Slide>
+        <SchemaTable name='Indexes' content={indexes} />
+        <SchemaTable name='Constraints' content={constraints} />
+        <br />
+        <p className='lead'>Execute the following query for more information</p>
```
Comment (oskarhane) [inline src/browser/modules/Stream/SchemaFrame.jsx]: The links doesn't really show _more_ information about the indexes and constraints, but rather semi-related info.
I would probably rewrite this to be something in the lines of: `Execute the following command to visualize what's related, and how` (we use that phrase in the sidebar "Saved scripts" -> "Data profiling").

File: src/browser/modules/Stream/SchemaFrame.jsx [hunk 6]
```diff
@@ -32,7 +35,7 @@ import {
   StyledBodyTr,
   StyledTd
 } from 'browser-components/DataTables'
-import { Directives } from 'browser-components/Directives'
+import Directives from 'browser-components/Directives'
```
Comment (nglgzz) [inline src/browser/modules/Stream/SchemaFrame.jsx]: Not sure why, but after I connected `SchemaFrame` to redux, the `onItemClick` function from `Directives` wouldn't work unless I imported the connected version.

**Pull request merged**